src/EnergyPlus/WindowManager.hh

```diff
@@ -388,213 +306,202 @@ namespace WindowManager {
+    void ViewFac(Real64 s,         // Slat width (m)
+                 Real64 h,         // Distance between faces of adjacent slats (m)
+                 Real64 phib,      // Elevation angle of normal to slat (radians)
+                 Real64 phis,      // Profile angle of radiation source (radians)
+                 Array2A<Real64> F // View factor array
     );
 
-    //*****************************************************************************************
-
     void InvertMatrix(EnergyPlusData &state,
                       Array2A<Real64> a, // Matrix to be inverted
                       Array2A<Real64> y, // Inverse of matrix a
                       Array1A_int indx,  // Index vector for LU decomposition
-                      int const np,      // Dimension of matrix
-                      int const n);
-
-    //*****************************************************************************************
+                      int np,            // Dimension of matrix
+                      int n);
 
     void LUDCMP(EnergyPlusData &state,
                 Array2A<Real64> A, // matrix
-                int const N,
-                int const NP,
+                int N,
+                int NP,
                 Array1A_int INDX,
                 int &D);
 
-    //*****************************************************************************************
-
-    void LUBKSB(Array2A<Real64> A, int const N, int const NP, Array1A_int INDX, Array1A<Real64> B);
+    void LUBKSB(Array2A<Real64> A, int N, int NP, Array1A_int INDX, Array1A<Real64> B);
 
     // added for custom solar or visible spectrum
-
     void CheckAndReadCustomSprectrumData(EnergyPlusData &state);
 
-    //*****************************************************************************************
-
     void initWindowModel(EnergyPlusData &state);
 
-    //*****************************************************************************************
-
 } // namespace WindowManager
 
 struct WindowManagerData : BaseGlobalStruct
 {
 
-    Real64 sigma;   // Stefan-Boltzmann constant
-    Real64 TKelvin; // conversion from Kelvin to Celsius
-    int nume;       // Number of wavelength values in solar spectrum
-    int numt3;      // Number of wavelength values in the photopic response
+    static Real64 constexpr sigma = 5.6697e-8;                         // Stefan-Boltzmann constant
+    static Real64 constexpr TKelvin = DataGlobalConstants::KelvinConv; // conversion from Kelvin to Celsius
+    static int constexpr nume = 107;                                   // Number of wavelength values in solar spectrum
+    static int constexpr numt3 = 81;                                   // Number of wavelength values in the photopic response
+
+    //                                      Dens  dDens/dT  Con    dCon/dT   Vis    dVis/dT Prandtl dPrandtl/dT
+    std::array<Real64, 8> const AirProps = {1.29, -0.4e-2, 2.41e-2, 7.6e-5, 1.73e-5, 1.0e-7, 0.72, 1.8e-3};
```

Comment:
There should be some enumeration for this where `DENS = 0`, `dDENSdT = 1`, etc.

Reply:
Yeah...or this shouldn't be an array.  I don't see what an ordered array does for these disparate physical properties.

Reply:
This feels more like a `struct` than an `array`.